src/tests/multihost/ipa/test_adtrust.py

```diff
@@ -607,3 +607,72 @@ def test_ad_user_ssh_ipa_client(multihost):
                                         '/var/log/sssd/krb5_child.log')
         multihost.client[0].run_command(f'grep -i "{log_str}" '
                                         f'/var/log/sssd/sssd_{domain_name}.log')
+
+    @staticmethod
+    def test_skip_members_in_view_search(multihost):
+        """
+        :title: Skip group members that point to an entry in the views base search
+        :id: 40adfa6a-b951-4aab-aa5b-57e149526e8d
+        :customerscenario: true
+        :bugzilla:
+          https://bugzilla.redhat.com/show_bug.cgi?id=2151403
+        :description: When resolving the nested groups, any entry in cn=views,cn=accounts,$BASEDN
+        (or whatever the user configured using ipa_views_search_base) must be ignored.
+        :setup:
+         1. Configure trust between IPA server and AD.
+         2. Configure client machine with SSSD integrated to IPA.
+        :steps:
+          1. Add a user1 ID override in 'default trust view'.
+          2. Create ext_group as a external group and add user in admins and external group.
+          3. Make ext_group a member of the admins group.
+          4. Run an "id" command for the override user.
+        :expectedresults:
+          1. Successfully added user1 in default trust view.
+          2. Successfully created and added user in admins and external group.
+          3. Made external_group a member of the admins group.
+          4. Id command succeeds, admin group is visible, groups are properly resolved.
+        """
+        domain = multihost.ad[0].domainname
+
+        ipa_client = sssdTools(multihost.client[0])
+        ipa_client.clear_sssd_cache()
+
+        # Here user1 is already added in AD and also member of group1, group2 and group3
```

Comment:
It might be a good idea to make sure of availability/create a kerberos ticket for admin on master[0], otherwise the test depends on some other test doing that and is prone to get broken at some point.